### PR TITLE
Parse Long from String in ConstantResult in Platform Node

### PR DESCRIPTION
**Pull request by abhishoya-gs**
#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/helpers/platform/DefaultExecutionNodeContext.java [hunk 1]
```diff
@@ -82,6 +82,10 @@ else if (clazz.equals(PureDate.class) && value instanceof EngineDate)
             {
                 return (T) ((EngineDate) value).transformToPureDate();
             }
+            else if (clazz.equals(Long.class) && value instanceof String)
+            {
+                return (T) Long.valueOf(Long.parseLong((String)value));
```
Comment (aziemchawdhary-gs) [inline legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/helpers/platform/DefaultExecutionNodeContext.java:87]: Do you need the call to Long.parseLong here? There is a Long.valueOf method that takes in a string and internally calls Long.valueOf on Long.parseLong

Comment (abhishoya-gs) [reply legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/helpers/platform/DefaultExecutionNodeContext.java:87]: Noted

File: legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/helpers/platform/DefaultExecutionNodeContext.java [hunk 2]
```diff
@@ -82,6 +82,10 @@ else if (clazz.equals(PureDate.class) && value instanceof EngineDate)
             {
                 return (T) ((EngineDate) value).transformToPureDate();
             }
+            else if (clazz.equals(Long.class) && value instanceof String)
```
Comment (hardikmaheshwari) [inline legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/nodes/helpers/platform/DefaultExecutionNodeContext.java:85]: This doesn't seem like the right place for the change. If clazz is Long value should not be string.
Possibly we should fix normalization logic here - https://github.com/finos/legend-engine/blob/master/legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/validation/FunctionParametersParametersValidation.java#L40

**Review by hardikmaheshwari — CHANGES_REQUESTED** [review]

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**